src/components/DepositWidget/Row.tsx

```diff
@@ -75,6 +163,20 @@ export const Row: React.FC<RowProps> = (props: RowProps) => {
   const [visibleForm, showForm] = useState<'deposit' | 'withdraw' | void>()
   const exchangeBalanceTotal = exchangeBalance.add(depositingBalance)
 
+  // Checks innerWidth
+  let showResponsive = innerWidth < 500
+
+  useMemo((): void => {
+    const bodyClassList: string[] | DOMTokenList = (window && window.document.body.classList) || []
+    const noScrollActive = Array.from(bodyClassList).find((className: string): boolean => className === 'noScroll')
```

Comment:
This is a bit wrong. If there's a window, there's a document and a body and classList is always available even if empty.
If there's no window, then `window` will throw a `RefrenceError` and the right check is `typeof window !=== 'undefined`

Consider instead
```ts
const noScrollActive = document.body.classList.contains('noScroll')
```
with window check if you want